### PR TITLE
Add new humidity control in humidifier more info

### PR DESCRIPTION
**Pull request by piitaya**
### Humidifier
CleanShot 2023-08-08 at 10 25 54

### Dehumidifier
CleanShot 2023-08-08 at 10 26 02

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by spacegaier** [comment]
This feels for the lack of a better word "a bit clunky",m since we have four different font variants on there (in therms of size and weight). Is there a way to harmonize that a bit? E.g. the "Current humidity" text feels too small compared to the rest and the card title.

**Review by bramkragten — DISMISSED** [review]

**Comment by spacegaier** [comment]
Is it really clear to the user which percentage value is target and which is current or would it help to add a "Current" label above the current value?

**Comment by piitaya** [comment]
> Is it really clear to the user which percentage value is target and which is current or would it help to add a "Current" label above the current value?

Yes I was thinking of that too.

**Comment by piitaya** [comment]
@spacegaier Tested some changes

CleanShot 2023-06-26 at 15 36 52

**Comment by spacegaier** [comment]
That is a good improvement 👍 

The only other thought I currently have is that to the user it might not directly be clear what curve element is the current and what is the target value (in this case blue bar vs white indicator).

E.g. when it sits next to a gauge card, where the colored bar represents the current value, where as in this card here the colored bar is the target and the white indicator is the current one which means a different mental user model.

**Comment by piitaya** [comment]
Feature postponed to next release because UX.UI needs to be discussed

File: src/dialogs/more-info/components/humidifier/ha-more-info-humidifier-humidity.ts [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+    this._callService();
+  }
+
+  private _valueChanging(ev: CustomEvent) {
+    const value = (ev.detail as any).value;
+    if (isNaN(value)) return;
+    this._targetHumidity = value;
+  }
+
+  private _debouncedCallService = debounce(() => this._callService(), 2000);
```
Comment (bramkragten) [inline src/dialogs/more-info/components/humidifier/ha-more-info-humidifier-humidity.ts:63]: 2 or 1 sec debounce?

File: src/dialogs/more-info/controls/more-info-humidifier.ts [hunk 2]
```diff
@@ -42,65 +50,74 @@ class MoreInfoHumidifier extends LitElement {
...
+      stateObj,
+      HumidifierEntityFeature.MODES
+    );
 
-    const rtlDirection = computeRTLDirection(hass);
+    const currentHumidity = this.stateObj.attributes.current_humidity;
 
     return html`
+      ${currentHumidity
+        ? html` <div class="current">
```
Comment (bramkragten) [inline src/dialogs/more-info/controls/more-info-humidifier.ts:62]:
```suggestion
        ? html`<div class="current">
```

**Review by bramkragten — DISMISSED** [review]

File: src/dialogs/more-info/components/humidifier/ha-more-info-humidifier-humidity.ts (hunk 1 above)
Comment (piitaya) [inline src/dialogs/more-info/components/humidifier/ha-more-info-humidifier-humidity.ts:63]:
```suggestion
  private _debouncedCallService = debounce(() => this._callService(), 1000);
```

**Review by bramkragten — DISMISSED** [review]

**Review by bramkragten — APPROVED** [review]

**Pull request merged**